src/base-styles/helperClasses/index.stories.mdx

```diff
@@ -20,6 +20,44 @@ export const positionStyles = {
 
 <Meta title="Style Utilities/Style Utilities" />
 
+## Responsive
+
+Select FDS classes can be prefixed to be applied at breakpoints. There are 4 mobile first breakpoints:
+
+<Story name="Responsive">
+  <>
+    <TableLayout
+      isCSS
+      headers={['Breakpoint prefix', 'Minimum width']}
+      rows={[
+        ['`s`', '480px'],
+        ['`m`', '768px'],
+        ['`l`', '1024px'],
+        ['`xl`', '1280px'],
+      ]}
+    />
+  </>
+</Story>
+
+The following classes can be prefixed and used responsively:
+
+- [margin & padding](/?path=/docs/style-utilities-style-utilities--margin-and-padding)
+- [align](?path=/docs/style-utilities-style-utilities--text-align)
+- [alignChild](?path=/docs/style-utilities-style-utilities--child-align)
+- [display](/?path=/docs/style-utilities-style-utilities--display)
+
+Example of usage:
+
+```
+<div className="padding--top--l lg:padding--top--xl" />
+```
+
+Guidelines / Notes:
+
+- Don't use `sm:` to target mobile devices
```

Comment:
```suggestion
- Don't use `s:` to target mobile devices
```

Reply:
This point might be a little misleading as it implies you can target viewports based on max-width, which you can't

Reply:
<img width="834" alt="Screen Shot 2021-04-06 at 1 59 02 PM" src="https://user-images.githubusercontent.com/708820/113764185-3fabda80-96e0-11eb-9131-d64eb8b616ee.png">

Would it help if we added some of the context in this picture that inspired this line?